INCLUDE/COM/MEN/smb2_shc.h

```diff
@@ -182,6 +182,14 @@ struct shc_configdata {
 	 *  UPS Minimum Run Charge in %
 	 */
 	u_int8 upsMinRunCharge;
+	/**
+	 * Enable power button persistence
+	 */
+	u_int8 persistent_pwrbtn_enabled;
```

Comment:
Variable name shall be persistentPwrbtnEnabled according Guideline for C programming.
Remark: When you change/extend existing code, it's ok to stick to the used name schema. But now it's inconsistent to the other ones e.g. upsMinRunCharge.